src/components/Todo.js

```diff
@@ -1,12 +1,13 @@
 import React, { useState } from 'react';
-import TodoForm from './TodoForm';
+import TodoList from './TodoList';
 import { RiCloseCircleLine } from 'react-icons/ri';
 
-function Todo({ todos, completeTodo, removeTodo, updateTodo }) {
+function Todo({ todos, completeTodo, removeTodo }) {
  
 
     return todos.map((todo, index) => (
         <div className={todo.isComplete ? 'todo-row complete' : 'todo-row'} key={index}>
+            <input type="checkbox" onChange={()=> {completeTodo(todo.id)}}/>
```

Comment:
Create a function and pass it to onChange and onClick.

Reply:
DONE